src/emysql_conn.erl

```diff
@@ -171,13 +171,9 @@ open_connections(Pool) ->
                 #emysql_connection{} = Conn ->
                     open_connections(Pool#pool{available = queue:in(Conn, Pool#pool.available)});
```

Comment:
Line 172 is too long:                     open_connections(Pool#pool{available = queue:in(Conn, Pool#pool.available)});.